debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleChangeEventSourceFactory.java

```diff
@@ -42,7 +48,20 @@ public SnapshotChangeEventSource getSnapshotChangeEventSource(OffsetContext offs
 
     @Override
     public StreamingChangeEventSource getStreamingChangeEventSource(OffsetContext offsetContext) {
-        return new OracleStreamingChangeEventSource(
+        OracleConnectorConfig.ConnectorAdapter adapter  =
+                OracleConnectorConfig.ConnectorAdapter.parse(config.getString(CONNECTOR_ADAPTER));
+        if (adapter == OracleConnectorConfig.ConnectorAdapter.XSTREAM) {
+            return new io.debezium.connector.oracle.xstream.OracleStreamingChangeEventSource(
```

Comment:
I'd suggest to name them `XstreamStreamingChangeEventSource` and `LogMinerStreamingChangeEventSource`.